frontendGraphApi/src/main/java/com/virtuslab/gitmachete/frontend/graph/api/paint/IColorGenerator.java

```diff
@@ -0,0 +1,7 @@
+package com.virtuslab.gitmachete.frontend.graph.api.paint;
+
+import java.awt.Color;
+
+public interface IColorGenerator {
```

Comment:
Nit: `IColorProvider`? `Generator` suggests there's more work needed to fetch the color (rather than just retrieve it from a static map).

Reply:
Plus ofc cascade the naming change down the codebase if you agree with the suggestion